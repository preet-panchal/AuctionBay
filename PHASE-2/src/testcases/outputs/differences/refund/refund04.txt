--- ./../testcases/outputs/expected/refund/refund04.txt	2023-02-13 15:21:18
+++ ./../testcases/outputs/actual/refund/refund04.txt	2023-03-05 16:32:33
@@ -13,13 +13,18 @@
 
 Enter operation:
 Enter username:
-Hello admin (AA)!
 
+Error: This user does not exist in the user-accounts file.
 Enter operation:
-Enter buyer's username:
-Enter seller's username:
-Enter amount to refund:
-Error: Invalid input for refund amount.
+Error: You must login first.
 
 Enter operation:
-admin01 successfully logged out.
\ No newline at end of file
+Error: You must login first.
+
+Enter operation:
+Error: You must login first.
+
+Enter operation:
+Error: You must login first.
+
+Enter operation:
