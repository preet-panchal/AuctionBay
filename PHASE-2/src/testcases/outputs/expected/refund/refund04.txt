Welcome to AuctionBay!
--
Here is a list of operations (what to input):
- Login (login)
- Create (create)
- Delete (delete)
- Advertise (advertise)
- Bid (bid)
- Refund (refund)
- Add Credit (addcredit)
- List All Items (listall)
- Logout (logout)

Enter operation:
Enter username:
Hello User01 (AA)!

Enter operation:
Enter buyer's username:
Enter seller's username:
Enter refund amount:
Error - amount must be a number.

Enter operation:
User01 successfully logged out.
